packages/autocertifier-client/src/exports.ts

```diff
@@ -0,0 +1,20 @@
+export { AutoCertifierClient, AUTOCERTIFIER_SERVICE_ID } from './AutoCertifierClient'
```

Comment:
Why we need to export this internal id?

Reply:
The id is used by both DHT, AutoCertifier and the AutoCertifierClient there has to be a single source to ensure that the same id is always used